src/openfermion/utils/_remove_symmetry_qubits_test.py

```diff
@@ -0,0 +1,171 @@
+""" Unit tests of remove_symmetry_qubits().
+    Test that it preserves the correct
+    ground state energy, and reduces the number
+    of qubits required by 2.
+"""
+
+import unittest
+
+import openfermion
+import openfermionpyscf
```

Comment:
these are awesome tests but unfortunately you cannot import openfermionpyscf inside of openfermion (but you can import openfermion inside of openfermionpyscf). There are many reasons for this. Among them: some people want to use openfermion without installing openfermionpyscf / pyscf (for instance, if you use windows where it is impossible to install pyscf). And we can't have tests failing for those users. So if you want to have tests that depend on openfermionpyscf, you'd need to put them in the openfermionpyscf repo, which you are welcome to do. Otherwise, perhaps you should hardcode the outputs so that these tests can run without openfermionpyscf. Thanks!